plutus-metatheory/src/Algorithmic/Reduction.lagda

```diff
@@ -1,43 +1,43 @@
 \begin{code}
-{-# OPTIONS --rewriting #-}
+--{-# OPTIONS --rewriting #-}
```

Comment:
I left it because you need it if you want to debug using `Debug.Trace`. Also commented in the file is
`--import Debug.Trace as Debug`

We can remove both of them, or leave them commented as an easier way of enabling debugging.

Reply:
I'll let @michaelpj decide.